nilearn/signal.py

```diff
@@ -508,6 +508,9 @@ def clean(signals, sessions=None, detrend=True, standardize=True,
                 signals -= confounds.dot(inv).dot(confounds.T).dot(signals)
 
     if low_pass is not None or high_pass is not None:
+        if t_r is None:
+            raise ValueError("Repetition time must be specified for filtering")
```

Comment:
Maybe I would say "Repetition time (t_r) must be specified..." to make it more explicit what the user must change.